employees/templates/employees/project_report_list.html

```diff
@@ -23,7 +23,10 @@ <h1>
 </h1>
 {% with date=path|extract_year_and_month_from_url %}
 <a href="{% url 'export-project-data-xlsx' pk=object.pk year=date.0 month=date.1 %}" class="btn btn-info">
-        Export Reports
+    Export Reports
```

Comment:
Please add here that this button gives `XLSX` export

Reply:
:+1: done